frontend/src/components/elements/PlotlyChart/PlotlyChart.tsx

```diff
@@ -37,7 +38,11 @@ class PlotlyChart extends React.PureComponent<Props> {
     const el = this.props.element
 
     const height: number =
-      el.get("height") > 0 ? el.get("height") : DEFAULT_HEIGHT
+      el.get("height") > 0
```

Comment:
Same as the above, we can't go fullscreen height if user supplies a height in the report?

Reply:
Yes, exactly like above, if the user set a value, we should use that value.
If you want, we can talk if this is the right behavior.